translations/base-de.yaml

```diff
@@ -348,8 +348,8 @@ ingame:
                 title: Unterstütze Mich
                 desc: Ich entwickle in meiner Freizeit!
             achievements:
-                title: Achievements
-                desc: Hunt them all!
+                title: Erfolge
```

Comment:
In der deutschen Version von Steam wird `Errungenschaften` verwendet, also würde ich es hier konsistent halten. Ausnahme: Es ist zu lang für das UI.

Reply:
Okay soll ich es ändern oder macht ihr das?

Reply:
Geändert